ui/src/main/resources/Meeting/WebHome.xml

```diff
@@ -48,7 +48,7 @@
   'leader': {"type":"text","size":10,"html":true},
   'place': {"type":"text","size":10},
   'startDate': {"type":"text","size":10},
-  'durationString': {"type":"text","filterable":true,"sortable":false,"html":true},
+  'durationString': {"type":"text","filterable":true,"sortable":true,"html":true},
```

Comment:
Apparently the main problem is that the sort doesn't work. Try to sort entries like: '4h 30 min', '30 min', '2h'